Apsitvarkom.Models/Mapping/CleaningEventProfile.cs

```diff
@@ -19,6 +19,12 @@ private void MapRequests()
     {
         CreateMap<ObjectIdentifyRequest, CleaningEvent>(MemberList.None)
             .ForMember(x => x.Id, opt => opt.MapFrom(x => x.Id));
+        CreateMap<CleaningEventUpdateRequest, CleaningEvent>()
+            .ForMember(x => x.Id, opt => opt.PreCondition(src => src.Id is not null))
```

Comment:
we already check this precondition in the validator. we are sure, that the `CleaningEventUpdateRequest` has an id, so I don't think is really necessary.
However, I also don't think this is a mistake, so whatever you decide is good [it's just that I've noticed the `PollutedLocationUpdateRequest` doesn't have this, while having stuff match is preferred]